x-pack/plugins/cases/public/components/case_view/case_view_page.tsx

```diff
@@ -38,67 +36,37 @@ export const CaseViewPage = React.memo<CaseViewPageProps>(
     caseId,
     fetchCase,
     onComponentInitialized,
-    updateCase,
     refreshRef,
     ruleDetailsNavigation,
     actionsNavigation,
     showAlertDetails,
     useFetchAlertData,
   }) => {
     const { userCanCrud, features } = useCasesContext();
-    const { metricsFeatures } = useCasesFeatures();
-    useCasesTitleBreadcrumbs(caseData.title);
-
     const { navigateToCaseView } = useCaseViewNavigation();
     const { urlParams } = useUrlParams();
+    const queryClient = useQueryClient();
+
+    useCasesTitleBreadcrumbs(caseData.title);
+
     const activeTabId = useMemo(() => {
       if (urlParams.tabId && Object.values(CASE_VIEW_PAGE_TABS).includes(urlParams.tabId)) {
         return urlParams.tabId;
       }
       return CASE_VIEW_PAGE_TABS.ACTIVITY;
     }, [urlParams.tabId]);
 
-    const [initLoadingData, setInitLoadingData] = useState(true);
     const init = useRef(true);
     const timelineUi = useTimelineContext()?.ui;
 
-    const getCaseUserActions = useGetCaseUserActions(caseId, caseData.connector.id);
-
-    const {
-      fetchCaseUserActions,
-      caseServices,
-      isLoading: isLoadingUserActions,
-    } = getCaseUserActions;
-
-    const refetchCaseUserActions = useCallback(() => {
-      fetchCaseUserActions(caseId, caseData.connector.id);
-    }, [caseId, fetchCaseUserActions, caseData]);
-
-    const {
-      metrics,
-      isLoading: isLoadingMetrics,
-      fetchCaseMetrics,
-    } = useGetCaseMetrics(caseId, metricsFeatures);
-
     const handleRefresh = useCallback(() => {
-      fetchCase();
-      fetchCaseMetrics();
-      refetchCaseUserActions();
-    }, [fetchCase, refetchCaseUserActions, fetchCaseMetrics]);
-
-    const handleUpdateField = useCallback(
-      (newCase: Case, _updateKey: UpdateKey) => {
-        updateCase({ ...newCase, comments: caseData.comments });
-        fetchCaseUserActions(caseId, newCase.connector.id);
-        fetchCaseMetrics();
-      },
-      [updateCase, caseData, fetchCaseUserActions, caseId, fetchCaseMetrics]
-    );
+      queryClient.invalidateQueries(CASE_VIEW_CACHE_KEY);
```

Comment:
All the caseview page cache keys have this one as a namespace. Invalidating this key will trigger all components in the page to refetch data.